test-containerd/instantiate_vm_and_test.sh

```
@@ -40,8 +40,12 @@ function delete_network() {
   # Ensure we are yet connected
   echo "" | ibmcloud login
   # Remove network after test
-  ibmcloud pi netd $1
-  return $?
+  NET_DEL_TIMEOUT=10
+  i=0
+  while [ "$i" -lt "$NET_DEL_TIMEOUT" ] &&  ! ibmcloud pi netd $1; do
+    i=$((i+1))
+    sleep 60
+  done
```

is there still a possibility this fails and we need to return `$?` ?

---

It can potentially fail to delete the network, but I'm not sure what the value of `$?` would be (or which previous command it would come from) if it was returned directly after the loop. Would it be better to compare `i` and `NET_DEL_TIMEOUT` after the loop and return a non-zero value if they are equal?

---

true, we should capture the actual rc from the netd command. just so we don't mask a failure here